elf/cmdline.cc

```diff
@@ -998,6 +1002,10 @@ std::vector<std::string> parse_nonpositional_args(Context<E> &ctx) {
     }
   }
 
+  ctx.arg.library_paths.insert(ctx.arg.library_paths.end(),
+          default_library_paths.begin(),
+          default_library_paths.end());
```

Comment:
I'm not fully getting what is difference between `-Y` and `-L` options -- at the end, we are just concatenating them.

Reply:
Oh, I see now.  It does make sense, thanks.